lib/pry/pry_class.rb

```diff
@@ -1,7 +1,17 @@
 require 'pry/config'
 class Pry
 
-  HOME_RC_FILE = ENV["PRYRC"] || "~/.pryrc"
+  HOME_RC_FILE = proc do
```

Comment:
Yes, I myself prefer the idea of class method (mostly for being easy to test). I'm just not sure whether you guys like it. If consensus is met, I'm more than happy to wrap it into a (private?) method.